agent_code/our_agent_v2/train.py

```diff
@@ -9,13 +9,18 @@
 matplotlib.use("Agg") # Non-GUI backend, needed for plotting in non-main thread.
 import matplotlib.pyplot as plt
 
+import warnings
+
+
 import settings as s
 import events as e
 from .callbacks import transform, fname, FILENAME, DR_BATCH_SIZE
 
 # Transition tuple. (s, a, r, s')
 Transition = namedtuple('Transition',
-                        ('state', 'action', 'next_state', 'reward'))
+                        ('state', 'action', 'next_state', 'reward', 'n_step_next_state'))
+
+warnings.filterwarnings("ignore", category=np.VisibleDeprecationWarning) 
```

Comment:
This should probably be solved, not suppressed.

Reply:
```suggestion
```